app/controllers/users_controller.rb

```diff
@@ -0,0 +1,46 @@
+class UsersController < ApplicationController
```

Comment:
I think we should only add the necessary methods for this story which is list users of an organization. I mean, the destroy method for example, is not necessary for now

Reply:
done